lib/posix/pthread.c

```diff
@@ -662,7 +694,10 @@ int pthread_join(pthread_t pthread, void **status)
 	/* other possibilities? */
 	__ASSERT_NO_MSG(err == 0);
 
+	LOG_DBG("Joined  pthread %x", pthread);
```

Comment:
```suggestion
	LOG_DBG("Joined pthread %x", pthread);
```

Reply:
That spacing is kind of intentional to get related messages to align vertically. I can remove if necessary though.